DataFormats/Common/interface/RefHolder_.h

```diff
@@ -7,6 +7,7 @@
 #include "FWCore/Utilities/interface/OffsetToBase.h"
 #include <memory>
 #include <typeinfo>
+#include <string>
```

Comment:
Why was this needed? `std::string` is not used anywhere in this file. What was the missing symbol needed?

Reply:
Never mind. I see `std::string&`. I shouldn't write comments when I haven't finished my coffee :(.